lib/scraping_hub.rb

```diff
@@ -8,13 +8,20 @@ module ScrapingHub
   def self.configure
     setup = ScrapingHub::Setup
     yield setup
```

Comment:
I think the setup inside the class is simpler, it doesn't need to have a hole class for this.